Game.IGame/IGame.cs

```diff
@@ -6,8 +6,8 @@ namespace Game.IGame
     public enum GameResults
     {
         InProgress = 0,
-        PlayerOneWins = 1,
-        PlayerTwoWins = 2,
+        CurrentPlayerWins = 1,
+        CurrentOpponentWins = 2,
```

Comment:
po co ta zmiana? wcześniej było jasne, że player1 to w warcabach biały, a w othello czarny, a teraz to jakieś pomieszane będzie

Reply:
Cofnąłem, ale teraz musisz zaimplementować w Othello dodatkową metodę z IGame